RubberduckTests/Symbols/ModuleBodyElementDeclarationTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor;
+using Rubberduck.VBEditor.SafeComWrappers;
+using RubberduckTests.Mocks;
+
+namespace RubberduckTests.Symbols
+{
+    [TestFixture]
+    public class ModuleBodyElementDeclarationTests
+    {
+        [Test]
+        [Category("Resolver")]
+        public void FindFindInterfaceMemberMatchesDeclarationTypes()
+        {
+            var intrface =
```

Comment:
trying to use keywords as variable names is really a bit annoying, innit?

how about `interfaceModule`?

Reply:
There might be some more of those. I'll try to change them as I find them.